sidecar/nginx/lua/amalgam8.lua

```diff
@@ -852,5 +857,24 @@ function Amalgam8:get_mytags()
    return self.mytags
 end
 
+function Amalgam8:get_myip()
+   return self.myip
+end
+
+function Amalgam8:get_env(name)
+   return os.getenv(name)
+end
+
+function Amalgam8:os_cmd(cmd, raw)
```

Comment:
Given that you are using this only to parse the IP address, you might as well rename this function to something like ip_cmd() and you just have to return the result of the "hostname -i" without the new lines.